ios/scripts/build_libsigner.sh

```diff
@@ -15,7 +16,8 @@ cd "$(dirname "${0}")/../../rust/signer"
 for i in "${IOS_ARCHS[@]}";
 do
   rustup target add "$i";
-  env -i PATH="${PATH}" \
+  env -i PATH="${PATH}" IPHONEOS_DEPLOYMENT_TARGET="${IOS_VERSION}" \
+  RUSTFLAGS="-C link-arg=-mios-version-min=${IOS_VERSION}" \
```

Comment:
Specified version in order to resolve problem:
```
object file (/Users/runner/work/parity-signer/parity-signer/rust/target/aarch64-apple-ios/release/deps/libzstd_sys-fd0631d9604c9610.rlib[5](44ff4c55aa9e5133-entropy_common.o)) was built for newer 'iOS' version (18.2) than being linked (10.0)
```
https://github.com/novasamatech/parity-signer/actions/runs/13386542020